### PR TITLE
Move content sets data to content manifest file

### PR DESCRIPTION
**Pull request by athos-ribeiro**
* CLOUDBLD-1003

Signed-off-by: Athos Ribeiro <athos@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request has a link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by athos-ribeiro** [comment]
Docs PR: https://github.com/containerbuildsystem/osbs-docs/pull/146

File: tests/plugins/test_add_content_sets.py [hunk 1]
```diff
@@ -61,66 +61,67 @@ def mock_content_sets_config(tmpdir, empty=False):
...
             CMD build /spam/eggs
-            LABEL some=40
+            LABEL name=eggs version=1.0 release=42
         """),
         dedent("""\
             FROM base_image
             CMD build /spam/eggs
-            ADD metadata_2.json /root/buildinfo/metadata_2.json
-            LABEL some=40
+            ADD content_manifest.eggs-1.0-42.json /root/buildinfo/content_manifests/eggs-1.0-42.json
```
Comment (lcarva) [inline tests/plugins/test_add_content_sets.py:77]: We want `/root/content_manifests/eggs-1.0-42.json` not `/root/buildinfo/content_manifests/eggs-1.0-42.json`, right?

Comment (lcarva) [reply tests/plugins/test_add_content_sets.py:77]: No. This is the right location as per the design doc.

**Comment by athos-ribeiro** [comment]
No release notes added. Feature introduced in this release cycle (already documented)

**Pull request merged**